src/AutoRest.CSharp/LowLevel/Output/DpgOutputLibraryBuilder.cs

```diff
@@ -40,24 +41,40 @@ public DpgOutputLibrary Build(bool isCadlInput)
                 .ToDictionary(ci => ci.Name);
 
             var topLevelClientInfos = SetHierarchy(clientInfosByName);
-
             var clientOptions = CreateClientOptions(topLevelClientInfos);
+
             SetRequestsToClients(clientInfosByName.Values);
 
+            var enums = new Dictionary<InputEnumType, EnumType>(InputEnumType.IgnoreNullabilityComparer);
+            var modelFactories = new Dictionary<InputModelType, Func<ModelTypeProvider>>();
+            var clients = new List<LowLevelClient>();
 
-            IReadOnlyDictionary<InputEnumType, EnumType> EnumFactory(TypeFactory typeFactory)
-                => isCadlInput ? _rootNamespace.Enums.ToDictionary(e => e, e => new EnumType(e, _defaultNamespace, "public", typeFactory, _sourceInputModel), InputEnumType.IgnoreNullabilityComparer) : new Dictionary<InputEnumType, EnumType>();
+            var library = new DpgOutputLibrary(enums, modelFactories, clients, clientOptions);
 
-            IReadOnlyDictionary<InputModelType, ModelTypeProvider> ModelsFactory(TypeFactory typeFactory)
-                => isCadlInput ? _rootNamespace.Models.ToDictionary(m => m, m => new ModelTypeProvider(m, typeFactory, _defaultNamespace, _sourceInputModel)) : new Dictionary<InputModelType, ModelTypeProvider>();
+            CreateEnums(enums, library.TypeFactory);
+            if (isCadlInput)
```

Comment:
For swagger input, where do we put the model?

Reply:
We don't support models for swagger input (at least for now).